pyproject.toml

```diff
@@ -32,6 +32,8 @@ classifiers = ['Development Status :: 1 - Planning']
 tests = ["pytest", "async-timeout", "pytest-asyncio", "pytest-cov", "scikit-image",]
 docs = ["jupyter-book"]
 lint = ["black", "flake8", "Flake8-pyproject", "flake8-pytest-style"]
+bw_demo = ["pyqtgraph", "mat73", "jax[cpu]", "proSVD",  
+           "bubblewrap@git+https://github.com/pearsonlab/Bubblewrap",]
```

Comment:
I think @draelos and I discussed having files inside the demos for dependencies (like a `requirements.txt` that one could `pip install` rather than add optional dependencies here for each demo. I think that's easier, and you can put instructions on installing demo dependencies in the readme for the demo.